.github/pull_request_template.md

```diff
@@ -0,0 +1,76 @@
+<!--  All Submissions -->
+
+
+## Community Note
+<!-- Please leave the community note as is. -->
+* Please vote on this PR by adding a :thumbsup: [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original PR to help the community and maintainers prioritize for review
+* Please do not leave "+1" or "me too" comments, they generate extra noise for PR followers and do not help prioritize for review
+
+
+## PR Checklist
+
+- [ ] I have followed the guidelines in our [Contributing Documentation](../blob/main/contributing/README.md).
+- [ ] I have checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change.
+- [ ] I have used a meaningful PR title to help maintainers and other users understand this change and help prevent duplicate work.
+Example:
+“`resource_name_here` - description of change e.g. adding property `new_property_name_here`”
```

Comment:
this might format better as
```suggestion
- [ ] I have used a meaningful PR title to help maintainers and other users understand this change and help prevent duplicate work. For example: “`resource_name_here` - description of change e.g. adding property `new_property_name_here`”
```
or
 ```suggestion
- [ ] I have used a meaningful PR title to help maintainers and other users understand this change and help prevent duplicate work. 
For example: “`resource_name_here` - description of change e.g. adding property `new_property_name_here`”
```